docs/src/pages/guides/cli-options.md

```diff
@@ -23,7 +23,7 @@ monika --config staging-set.json
 Configuration files may be placed remotely which you can specify using the same flag and using a URI.
 
 ```bash
-monika -c https://raw.githubusercontent.com/hyperjumptech/monika/main/config_sample/config.desktop.example.json
+monika -c https://raw.githubusercontent.com/hyperjumptech/monika/main/config_sample/config.desktop.example.yaml
```

Comment:
is it `.yaml` or `.yml`? seems like file is using `.yml`

Reply:
good question, and the internet is split haha.. the specification is YAML, so officialy yAml. But extensions are *.yml and *.yaml. Google seach actually returns more hits with *.yml (21.7 million) versus yaml (11.8 million).. Monika supports both extenion. The docs has been made more consistent to use *.yml